.buildkite/hooks/pre-command

```diff
@@ -65,6 +65,7 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats" || "$BUILDKITE_PIPELINE_SLUG" == "fi
 fi
 
 ENABLED_BEATS_PIPELINES_SLUGS=(
+  "beats"
```

Comment:
Do we need any of those env vars or secrets since the job triggers the other pipelines?

Reply:
same comment from me, let's remove this because it's not useful in anyway for the execution of the mainpipeline, via the monorepo plugin.

Reply:
For the `pre-exit` hook. It uses `util.sh` and it needs `BIN` var